app/views/any_not_listed/edit.html.erb

```diff
@@ -21,11 +21,11 @@
 <% content_for :card_footer do %>
   <%= fields_for(:form, @form, builder: Cfa::Styleguide::CfaFormBuilder) do |f| %>
     <%= f.button name: "form[any_not_listed]", value: "yes", class: "button" do %>
-      <i class="button__icon--left icon-check"></i>Yes
```

Comment:
For these buttons, there are some emoji included in the styleguide that we may want to use instead of creating new classes. I was working in the same area to pull out the styleguide component, so have done that work on the WIP branch: https://github.com/codeforamerica/alaska-fee-agent-assister/pull/25/files#diff-7253fcd6f980ee02395d571d713090c5R34

I think we should skip this change and the classes for now, since it'll be taken care of with the refactor.